iroh/src/run.rs

```diff
@@ -85,8 +85,7 @@ enum Commands {
 impl Cli {
     pub async fn run(&self) -> Result<()> {
         let config_path = iroh_config_path(CONFIG_FILE_NAME)?;
-        // TODO(b5): allow suppliying some sort of config flag. maybe --config-cli?
-        let sources = vec![Some(config_path.as_path())];
+        let sources = vec![self.cfg.as_deref(), Some(config_path.as_path())];
```

Comment:
huh, all the other commands line interfaces seem to order this the other way around.